sample-compose/app/src/androidTest/java/co/nimblehq/sample/compose/test/TestDispatchersProvider.kt

```diff
@@ -0,0 +1,21 @@
+package co.nimblehq.sample.compose.test
+
+import co.nimblehq.sample.compose.util.DispatchersProvider
+import kotlinx.coroutines.*
+import kotlinx.coroutines.test.TestDispatcher
+import kotlinx.coroutines.test.UnconfinedTestDispatcher
+
+@OptIn(ExperimentalCoroutinesApi::class)
+object TestDispatchersProvider : DispatchersProvider {
```

Comment:
Just curious, any benefit of having this as `object` instead of `class` ? 🙏

Reply:
@Tuubz we simply need [the implementation of a testDispatcherProvider with `UnconfinedTestDispatcher()`](https://github.com/nimblehq/android-templates/pull/384/files#diff-cab9096f29fe6421722b12a1bd2648e6e94656bee96783f0b92583eec9398997R14-R24) as default. We can convert this to be a full `CoroutineTestRule` when needed but not this time. An `object` with default `val testDispatcher: TestDispatcher = UnconfinedTestDispatcher()` is enough for this sample.